test/Microsoft.Azure.Devices.Edge.Test/PlugAndPlay.cs

```diff
@@ -63,29 +66,74 @@ public async Task DeviceClient()
                     DateTime seekTime = DateTime.Now;
                     await leaf.SendEventAsync(token);
                     await leaf.WaitForEventsReceivedAsync(seekTime, token);
-                    await this.Validate(this.iotHub.Hostname, leafDeviceId, TestModelId);
+                    await this.ValidateDevice(leafDeviceId, TestModelId);
                 },
                 async () =>
                 {
                     await leaf.DeleteIdentityAsync(token);
                 });
         }
 
-        public async Task Validate(string hostName, string deviceId, string expectedModelId)
+        [Test]
+        public async Task ModuleClient()
+        {
+            CancellationToken token = this.TestToken;
+            string loadGenImage = Context.Current.LoadGenImage.Expect(() => new ArgumentException("loadGenImage parameter is required for Priority Queues test"));
+            EdgeDeployment deployment = await this.runtime.DeployConfigurationAsync(
+                builder =>
+                {
+                    builder.GetModule(ModuleName.EdgeHub).WithEnvironment(new[] { ("UpstreamProtocol", "Mqtt") });
+                    builder.AddModule(LoadGenModuleName, loadGenImage)
+                    .WithEnvironment(new[]
+                    {
+                            ("testStartDelay", "00:00:00"),
+                            ("messageFrequency", "00:00:00.5"),
+                            ("transportType", Client.TransportType.Mqtt.ToString()),
+                            ("modelId", TestModelId)
+                    });
+                },
+                token);
+            EdgeModule filter = deployment.Modules[LoadGenModuleName];
+            await filter.WaitForEventsReceivedAsync(deployment.StartTime, token);
+            await this.ValidateModule(this.runtime.DeviceId, LoadGenModuleName, TestModelId);
+        }
+
+        async Task ValidateModule(string deviceId, string moduleId, string expectedModelId)
+        {
+            string requestString = $"https://{this.iotHub.Hostname}/twins/{deviceId}/modules/{moduleId}?api-version=2020-05-31-preview";
```

Comment:
Yeah it's a bit weird and I guess we could standardize it by having both be the twins call, but to me the digitaltwins call is more correct, since it's more specific. 

Sort of a moot point, since once the service sdk comes out with a way to see the modelId, we'll use that instead